docs/modules/pipelines/pages/compute-isolation.adoc

```diff
@@ -0,0 +1,25 @@
+= Jet Job Placement
+:description: Your Jet processing jobs can be distributed across a defined subset of the cluster. This approach provides finer control of your Jet processing, which means that you can distribute your workload to meet your requirements. 
+:page-enterprise: true
+
+{description}
+
+NOTE: Your license key must include `Advanced Compute` to activate this feature.
+
+For example, you can configure Jet processing jobs so that they run on lite members only, allowing you to split your computational and storage requirements without the need to configure each job separately. You set the members to use for your Jet job processing on a job-by-job basis.
+
+Distributing the processing job in this way allows you to find the best balance for your processing and storage requirements. Separating the processing from the data serving requirement means that less stress is put on your storage component's resources, as they only need to serve the data and not carry out any of the processing. This can help you to spread the load on your cluster across the members.
+
+Your storage components still use their resources to serve data. You must be sure that the processing element of the job uses considerably more resources than the data retrieval element before using isolated jobs. 
+
+To use job placement, create and/or submit a job using the `JobBuilder` API, which you can use to configure the following:
```

Comment:
```suggestion
For example, you can configure Jet processing jobs so that they run on lite members only, allowing you to split your computational and storage requirements without the need to configure each job separately. You control the members to use for your Jet job processing on a job-by-job basis.

Distributing the processing job in this way allows you to find the best balance for your processing and storage requirements. Separating the processing from the data serving requirement means that less stress is put on your storage component's resources, as they only need to serve the data and not carry out any of the processing. This can help you to spread the load on your cluster across the members.

Your storage components still use their resources to serve data. You must be sure that the processing element of the job uses considerably more resources than the data retrieval element before using job placement control in this manner. 

To use job placement control, create and/or submit a job using the `JobBuilder` API, which you can use to configure the following:
```

Reply:
Updated